script/service-worker.js.tmpl

```diff
@@ -2,17 +2,25 @@ self.addEventListener("push", function(event) {
   var data;
   if (event.data) {
     data = event.data.json();
-    event.waitUntil(self.registration.showNotification(data.title, data));
+    event.waitUntil(
+      self.registration.showNotification(data.title, data)
+      .then(function(notification){
+        firePushCallback({type: "push", tag: data.tag}, data.data.jwt);
```

Comment:
When exactly does this fire? When the notification is swiped away? When it is shown ?

Is this to indicate that the notification arrived at the phone?